document/JS.bs

```diff
@@ -0,0 +1,935 @@
+<pre class='metadata'>
+Title: WebAssembly JS API
+H1: WebAssembly
+Shortname: wasmjs
+Group: WebAssembly CG
+Status: w3c/CG-DRAFT
+Level: 1
+ED: https://webassembly.github.io/spec/JS.html
+Editor: WebAssembly Community Group
+Repository: WebAssembly/spec
+Abstract: This document provides an explicit JavaScript API for interacting with WebAssembly.
+Markup Shorthands: css no, markdown yes
+Ignored Terms: h1, h2, h3, h4, h5, h6, xmp
+</pre>
+
+<pre class='biblio'>
+{
+  "ECMA-262": {
+    "href": "https://tc39.github.io/ecma262",
+    "title": "ECMAScript® 2018 Language Specification",
+    "publisher": "ECMA TC39",
+    "status": "Current Editor's Draft"
+  },
+  "WEBASSEMBLY": {
+    "href": "https://webassembly.github.io/spec/",
+    "title": "WebAssembly Specification",
+    "publisher": "W3C WebAssembly Community Group",
+    "status": "Draft"
+  }
+}
+</pre>
+
+<pre class="anchors">
+urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
+    type: interface; for: ECMAScript
+        text: DataView; url: sec-dataview-objects
+        text: Map; url: sec-map-objects
+        text: Promise; url: sec-promise-objects
+        text: Set; url: sec-set-objects
+        text: SharedArrayBuffer; url: sec-sharedarraybuffer-objects
+    type: exception; for: ECMAScript
+        text: Error; url: sec-error-objects
+        text: NativeError; url: sec-nativeerror-constructors
+    type: dfn
```

Comment:
You're getting hit heavily by https://github.com/tabatkins/bikeshed/issues/809 and https://github.com/tabatkins/bikeshed/issues/861 here. Not much you can do right now, just a heads-up.